lib/editor/components/GtfsEditor.js

```diff
@@ -18,19 +18,68 @@ import {getConfigProperty} from '../../common/util/config'
 import {getEntityName, getTableById} from '../util/gtfs'
 import {GTFS_ICONS} from '../util/ui'
 
-import type {Feed, Project} from '../../types'
+import type {
+  ControlPoint,
+  Coordinates,
+  EditorTableData,
+  Entity,
+  Feed,
+  FeedInfo,
+  FetchStatus,
+  GtfsStop,
+  LatLng,
+  Pattern,
+  PatternStop,
+  Project
+} from '../../types'
 import type {UserState} from '../../manager/reducers/user'
+import type {EditorStatus} from '../reducers/data'
+import type {MapState} from '../reducers/mapState'
+import type {EditSettingsState} from '../reducers/settings'
+import type {EditorValidationIssue} from '../util/validation'
 
 type Props = {
+  subEntityId: number,
+  currentPattern: Pattern,
+  editSettings: EditSettingsState,
+  mapState: MapState,
+  feedSource: Feed,
+  feedInfo: FeedInfo,
+  zoomToTarget: ?number,
+  entities: Array<Entity>,
+  tripPatterns: Array<Pattern>,
+  fetchTripPatterns: () => void,
+  updateMapSetting: ({bounds?: any, target?: number, zoom?: number}) => void,
+  removeControlPoint: () => void,
+  updateUserMetadata: (any, any) => void,
+  updatePatternStops: (Pattern, Array<PatternStop>) => void,
+  handleControlPointDrag: () => void,
+  handleControlPointDragEnd: () => void,
+  handleControlPointDragStart: () => void,
+  setActivePatternSegment: () => void,
```

Comment:
let's alphabetize these.  Try using [this plugin](https://atom.io/packages/sorter).